src/LiveComponent/src/Resources/doc/index.rst

```diff
@@ -337,6 +337,22 @@ it won't, yet, make an Ajax call to re-render the component. Whenever
 the next re-render *does* happen, the updated ``max`` value will be
 used.
 
+
+Validating the form before triggering the action
```

Comment:
This documentation is not really about validating before triggering an action. It is about validating before updating the model, right? You DID also make it possible to use `validate` with the action modifier which is related to forms submitting, but that's not what you have here :)